index.md

```diff
@@ -2,7 +2,39 @@
 layout: default
 ---
 
-# Specifications
+# Introduction
+
+Matroska aims to become THE standard of multimedia container formats. It was derived from a project called MCF, but differentiates from it significantly because it is based on EBML (Extensible Binary Meta Language), a binary derivative of XML. EBML enables the Matroska Development Team to gain significant advantages in terms of future format extensibility, without breaking file support in old parsers.
+
+First, it is essential to clarify exactly "What an Audio/Video container is", to avoid any misunderstandings:
+
+    It is NOT a video or audio compression format (video codec)
+    It is an envelope for which there can be many audio, video and subtitles streams, allowing the user to store a complete movie or CD in a single file.
+
+Matroska is designed with the future in mind. It incorporates features you would expect from a modern container format, like:
+
+    Fast seeking in the file
+    Chapter entries
+    Full metadata (tags) support
+    Selectable subtitle/audio/video streams
+    Modularly expandable
+    Error resilience (can recover playback even when the stream is damaged)
+    Streamable over the internet and local networks (HTTP, CIFS, FTP, etc)
+    Menus (like DVDs have)
+
+Matroska is an open standards project. This means for personal use it is absolutely free to use and that the technical specifications describing the bitstream are open to everybody, even to companies that would like to support it in their products. The source code of the libraries developed by the Matroska Development Team is licensed under GNU L-GPL. In addition to that, there are also free parsing and playback libraries available under the BSD license, for commercial software and Hardware adoption.
+
+The Matroska has the following goals:
```

Comment:
actually I removed this line anyhow in a subsequent commit. It is here because I wanted to start from the original/official language and change from there.